src/_filter/collection/where.js

```diff
@@ -12,6 +12,9 @@ angular.module('a8m.where', [])
   .filter('where', function() {
     return function (collection, object) {
 
+      if(isUndefined(collection)) {
+        return false;
+      }
```

Comment:
I think the return value should be consistent with the other filters, e.g: [groupBy](https://github.com/a8m/angular-filter/blob/master/src/_filter/collection/group-by.js#L16-L18), [uniq](https://github.com/a8m/angular-filter/blob/master/src/_filter/collection/unique.js#L23-L25), etc..
also it will be great if you add test for this too.
thx @rodolfojnn